generate_build_config.py

```diff
@@ -26,30 +27,41 @@
 
 # Perform the build
 - /home/ubuntu/launchpad-buildd/mount-chroot $BUILD_ID
-- /home/ubuntu/launchpad-buildd/update-debian-chroot $BUILD_ID
+- /home/ubuntu/launchpad-buildd/update-debian-chroot $BUILD_ID{}
```

Comment:
I think we could just have this on a newline, which would make the code below a bit neater.

```
In [5]: yaml.load("""\
   ...: x:
   ...: - foo
   ...: 
   ...: - bar""")
Out[5]: {'x': ['foo', 'bar']}
```